src/main/java/com/it4045/pokemonviewer/enterprise/PokemonViewerController.java

```diff
@@ -16,13 +16,16 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
+import java.util.logging.Logger;
 
 @Controller
 public class PokemonViewerController {
 
     @Autowired
     IPokemonService pokemonService;
 
+   // Logger log = (Logger) LoggerFactory.getLogger(this.getClass());
+
```

Comment:
Avoid committing commented-out code to version control.  Version control offers better alternatives:
1) If you want to see what used to be there, you can diff branches to see history.
2) If you are working on an experimental feature, use a feature branch.

Commented-out code is clutter, which is technical debt.